Ladybird/Qt/main.cpp

```diff
@@ -103,6 +119,20 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     args_parser.add_option(expose_internals_object, "Expose internals object", "expose-internals-object");
     args_parser.parse(arguments);
 
+    WebView::SingletonProcessState singleton_process_state;
+    auto new_window = false;
+    if (TRY(singleton_process_state.connect(raw_urls, new_window)) == WebView::SingletonProcessState::ProcessDisposition::ExitProcess) {
+        outln("Opening in existing process");
+        return 0;
+    }
+
+    singleton_process_state.on_new_tab = [](auto const& urls) {
+        dbgln("asked to open new tab with urls: {}", urls);
+    };
```

Comment:
Seems like this shouldn't be here? We set it below

Reply:
fair. these got moved around a bit in the commits as data was available earlier/later.